kafl_fuzzer/common/config.py

```diff
@@ -247,22 +248,31 @@ def _parse_with_config(self, parser):
 
         config = confuse.Configuration('kafl', modname='kafl_fuzzer')
 
-        # check default config search paths
+        # check default config locations
         config.read(defaults=True, user=True)
 
+        # ENV based config
+        if 'KAFL_CONFIG_FILE' in os.environ:
+            try:
+                debug_print(f"Checking configuration at KAFL_CONFIG_FILE: {os.environ['KAFL_CONFIG_FILE']}")
+                config.set_file(os.environ['KAFL_CONFIG_FILE'])
+            except Exception as e:
```

Comment:
Your exception handling catch should be as specific as possible.
`except KeyError as e` here

Reply:
This is about file access and parse errors. Perhaps we want to catch `confuse.Exception` https://confuse.readthedocs.io/en/latest/api.html?highlight=set_file#module-confuse.exceptions